modules/openxr/openxr_api.cpp

```diff
@@ -271,7 +271,11 @@ bool OpenXRAPI::create_instance() {
 
 	XrInstanceCreateInfo instance_create_info = {
 		XR_TYPE_INSTANCE_CREATE_INFO, // type
+#ifdef ANDROID_ENABLED
```

Comment:
This needs to be solved differently, see other places where we extend structures through extensions.

Reply:
I think the correct way to handle this would be to extend `OpenXRExtensionWrapper` with a `void *set_instance_create_info_and_get_next_pointer(void *p_next_pointer)` method, which should be called here, and `OpenXRAndroidExtension` should add its entry to the struct chain by overriding that method.

Reply:
Yes exactly, it needs to work in the same way that we extend other structures, indeed with a `set_instance_create_info_and_get_next_pointer`. I thought we had this already in Godot 3 so I must have forgotten to port it over when I did this in Godot 4.